### ISSUE TITLE
Make Downloading Multiple Tiles Easier

### ISSUE THREAD

**Issue by stdavis** [agrc/raster#165]
This was @jacobdadams attempt many years ago: https://github.com/agrc/quilter

Maybe there's a way to solve this pain point within the app itself in the upcoming rewrite.

**Comment by jacobdadams** [comment]
Yeah, if we can generate a csv or other list of download links I think that would go far. There are download clients that can operate over a csv, and then once the user has got them extracted, Pro and other tools can handle mosaic datasets fairly well so we wouldn't necessarily need to do the merging process for them.

**Comment by stdavis** [comment]
I like the idea of adding a single button that would generate a list of tile URLs to ingest into some 3rd party downloader app.

**Comment by rkelson** [comment]
I absolutely agree. 

I typically use a script to concatenate a text list composed of path/tile/extension from a selected set of a tile index then punch it through a wget loop. Fancy I know ;)

**Comment by steveoh** [comment]
> I typically use a script to concatenate a text list composed of path/tile/extension from a selected set of a tile index then punch it through a wget loop.

We can do like we do on the website and offer multiple ways to use the data and wget being one of them since it is a common/popular utility.

**Comment by rkelson** [comment]
In my world I think it best to simply generate a list with path/tile/ext on separate lines. Then provide links to code samples for how to use the list.

For example I use wget a couple different ways but I provide an output directory parameter when I use it. I suppose these generated lists could create something like a .bat file you simply copy into a folder and run but it might be best to simply provide a list of tiles the user then decides on how to use.

**Comment by jacobdadams** [comment]
Couple of random thoughts:

- The CLI copy works well for small lists, but if there's a lot of tiles, it could become unwieldly. A .txt download option would be better for that.
- The curl command works in old school cmd.exe, so maybe we change "Power Shell" to just "Windows" in the tip
- I appreciate the concern about egress charges. Maybe we turn this on and watch our egress charges over the next six months. If it's a concern, we could roll it back (or maybe limit the number of tiles we generate a list for?)
- It would be really cool to be able to select multiple datasets to generate the list from (ie, if the AOI covers two different projects, so you could select the DEMs from one and from the other as well). It's only one or two extra steps saved, but it'd remove friction.
